### PR TITLE
New Autodesk Maya Cheatsheet

### PR DESCRIPTION
**Pull request by gautamkrishnar**
Display useful Autodesk Maya keyboard shortcuts

-----
IA Page: https://duck.co/ia/view/maya_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata post coming soon

File: share/goodie/cheat_sheets/json/maya.json [hunk 1]
```diff
@@ -0,0 +1,572 @@
+{
+  "id": "maya_cheat_sheet",
+  "name": "Autodesk Maya",
+  "description": "A list of useful Autodesk Maya keyboard shortcuts",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/maya.json]: @gautamkrishnar Lets change it to "Keyboard shortcuts for  Autodesk Maya, a 3D computer graphics software"

File: share/goodie/cheat_sheets/json/maya.json [hunk 2]
```diff
@@ -0,0 +1,572 @@
...
+    ],
+    "Playback Control": [
+      {
+        "val": "Move forward one frame",
+        "key": "[Alt] + [.]"
+      },
+      {
+        "val": "Move backward one frame",
+        "key": "[Alt] + [,]"
+      },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/maya.json]: @gautamkrishnar lets remove the + in between the keys

File: share/goodie/cheat_sheets/json/maya.json [hunk 3]
```diff
@@ -0,0 +1,572 @@
...
+        "key": ","
+      },
+      {
+        "val": "Turn Playback on/off",
+        "key": "[Alt] + [v]"
+      },
+      {
+        "val": "Go to Min Frame",
+        "key": "[Alt]+ [Shift] + [v]"
+      }
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/maya.json]: @gautamkrishnar same here

File: share/goodie/cheat_sheets/json/maya.json [hunk 4]
```diff
@@ -0,0 +1,572 @@
...
+      },
+      {
+        "val": "Show > Isolate Select > View SelectedDecelerate",
+        "key": "[Shift] + [I]"
+      }
+    ],
+    "Tool Commands": [
+      {
+        "val": "Complete current tool",
+        "key": "[Return]"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/maya.json]: @gautamkrishnar no need of [ ] here

File: share/goodie/cheat_sheets/json/maya.json [hunk 5]
```diff
@@ -0,0 +1,572 @@
...
+        "val": "Abort current tool",
+        "key": "~"
+      },
+      {
+        "val": "Enter tool Edit mode",
+        "key": "Insert"
+      },
+      {
+        "val": "Select Tool",
+        "key": "[Shift] [menu} + [Q]"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/maya.json]: @gautamkrishnar  the bracket for menu has a typo

File: share/goodie/cheat_sheets/json/maya.json [hunk 6]
```diff
@@ -0,0 +1,572 @@
...
+      {
+        "val": "Tangent marking menu",
+        "key": "[Shift] + [S] + [MMB]"
+      },
+      {
+        "val": "Set key for Rotate",
+        "key": "[Shift] + [E]"
+      },
+      {
+        "val": "Ser key for Scale",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/maya.json]: @gautamkrishnar typo

File: share/goodie/cheat_sheets/json/maya.json (hunk 1 above)
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/maya.json]: sounds good ;+1:

**Comment by gautamkrishnar** [comment]
@jophab you forgot to mention this:
best way to express arrow keys is directly using arrow ASCII characters (←, ↑, →, ↓)

**Comment by moollaza** [comment]
@gautamkrishnar please make an IA page for this PR and link it

**Comment by moollaza** [comment]
It should automatically be deployed to Beta for testing then :+1:

**Comment by moollaza** [comment]
At first glance this is looking a little long -- can we slim it down? Is all of this essential?

**Comment by gautamkrishnar** [comment]
@moollaza still having the same problem
screenshot 140

**Comment by moollaza** [comment]
>@moollaza still having the same problem

@gautamkrishnar you're seeing that because there is already an IA page :) -- we're updating the messaging to be more clear

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Maya Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/maya_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/maya.json [hunk 7]
```diff
@@ -0,0 +1,351 @@
...
+        "val": "Select last used tool (Excluding Select. Move. Rotate and Scale)",
+        "key": "y"
+      },
+      {
+        "val": "Snap Move. Rotate. Scale tool",
+        "key": "j"
+      },
+      {
+        "val": "Increase manipulator size",
+        "key": "[=] or []"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/maya.json]: Looks like there's a missing character here? What belongs inside the `[]`?

**Comment by moollaza** [comment]
@gautamkrishnar one small bug fix and then this is good!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Maya Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/maya_cheat_sheet_mobile.png)

**Comment by gautamkrishnar** [comment]
@moollaza done :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Maya Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/maya_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/maya.json [hunk 8]
```diff
@@ -0,0 +1,351 @@
...
+        "val": "Mask marking menu",
+        "key": "[q] [LMB]"
+      },
+      {
+        "val": "Move tool",
+        "key": "w"
+      },
+      {
+        "val": "Move tool marking menu",
+        "key": "[w]  [LMB]"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/maya.json]: @gautamkrishnar I think there is an extra space here :smile:

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/maya.json]: oops ok :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Maya Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/maya_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@gautamkrishnar thanks! This LGTM :+1:

**Pull request merged**